utils/cp_utils.py

```diff
@@ -60,14 +60,17 @@ def run_cellprofiler(
         analysis_run (bool, optional): will use functions to complete an analysis pipeline (default is False)
 
     """
-    print(f"Starting CellProfiler run on {pathlib.Path(path_to_images).name}")
-    # A log file is created for each plate or data set name (based on folder name with images)
-    with open(pathlib.Path(f"logs/cellprofiler_output_{pathlib.Path(path_to_images).name}.log"), "w") as cellprofiler_output_file:
-        # run CellProfiler for a illumination correction pipeline
-        command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}"
-        subprocess.run(command, shell=True, stdout=cellprofiler_output_file, stderr=cellprofiler_output_file)
-    cellprofiler_output_file.close()
+    # run CellProfiler illumination correction pipeline
+    if not analysis_run:
+        print(f"Starting CellProfiler run on {pathlib.Path(path_to_images).name}")
+        # A log file is created for each plate or data set name (based on folder name with images)
+        with open(pathlib.Path(f"logs/cellprofiler_output_{pathlib.Path(path_to_images).name}.log"), "w") as cellprofiler_output_file:
+            # run CellProfiler for a illumination correction pipeline
+            command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}"
+            subprocess.run(command, shell=True, stdout=cellprofiler_output_file, stderr=cellprofiler_output_file)
```

Comment:
Should `capture_output` be set here?

Reply:
Not going to lie, the documentation did not make it very clear what to use here, be it capture_output or not. I can set it as `capture_output=cellprofiler_output_file` since it should do the same thing based on the documentation.